data/data/agent/systemd/units/agent-extract-tui.service

```diff
@@ -1,9 +1,10 @@
 [Unit]
 Description=Extract agent-tui at boot
 After=selinux.service start-local-registry.service
-Before=agent-interactive-console.service
-ConditionPathExists=/etc/assisted/rendezvous-host.env
+ConditionPathExists=/etc/assisted/rendezvous-host.env.template
 ConditionPathExists=/etc/assisted/interactive-ui
+ConditionPathExists=!/usr/local/bin/agent-tui
```

Comment:
IIUC, this condition is required to allow enabling by the default the `agent-extract-tui.service`, so that it will not conflict during the regular workflow (which takes care of extracting the agent-tui artifacts from the initrd) - with the final intention of simplifying the unconfigured-ignition asset? If so a small comment could be useful here

Reply:
At least for me it was not immediately evident, since the service was expected to run only once (and currently only when the UI is present).  And btw, I think in future we should change the triggering condition for this service, detaching completely  from the UI and binding it to a more explicit one (ie something like `/etc/assisted/no-registry`) indicating that the payload is immediately available for consumption (and the same could be used by the TUI to skip the release check).
Yeah the commit message was useful (previous link seems broken now, I think it's this [one](https://github.com/openshift/installer/pull/9941/commits/00f6ab7e623ca3556ec97407cc7a8e3e8b0e88e9)), but a comment will be more evident IMHO